README.md

```diff
@@ -90,6 +90,27 @@ A fix for this has been proposed upstream at https://github.com/tailwindlabs/tai
 apk add build-base gcompat
 ```
 
+### Using asset-pipeline assets
+
+To use assets from the asset pipeline one would normally use `asset-url`. This doesn't work with Tailwind, as the CSS is not processed with sass. Instead use `url(image.svg)`, [since Sprockets v3.3.0](https://github.com/rails/sprockets-rails/pull/476) the URL will automatically be rewritten. In Tailwind this looks like this:
```

Comment:
`the URL will automatically be rewritten` for me as a reader this is a bit too much abstract, can we expand a bit on this so I don't have to open PR link to understand what rewritten means.